### PR TITLE
Adding recipe for zenodo-backpack

### PR DESCRIPTION
**Pull request by chklovski**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/zenodo-backpack/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - requests
+    - tqdm
+
+test:
+  imports:
+    - zenodo_backpack
+
+about:
+  home: "https://github.com/centre-for-microbiome-research/zenodo_backpack"
+  license: GPL3
```
Comment (dopplershift) [inline recipes/zenodo-backpack/meta.yaml:34]: Not sure why the linter okayed this, but it should be:

```suggestion
  license: GPL-3.0-or-later
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a minor tweak to the license metadata and this is good to go.

File: recipes/zenodo-backpack/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - requests
+    - tqdm
+
+test:
```
Comment (xhochy) [inline recipes/zenodo-backpack/meta.yaml:28]: Please add a call to `pip check` to check for the integrity with of the requirements specification.
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/zenodo-backpack/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5843a04b353ae613b474932c2a12d1fbfd529674074e73927ddc131eba7f1da8
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - conda=conda.cli.pip_warning:main
```
Comment (xhochy) [inline recipes/zenodo-backpack/meta.yaml:16]: This should not be listed here.
```suggestion
    - conda=conda.cli.pip_warning:main
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Review by xhochy — APPROVED** [review]

**Pull request merged**